.changeset/ninety-games-bake.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Is there a way I can open a integration test PR in dotcom, similar to primer/react? Or, if you have any suggestions to safely rolling this out, let me know!